web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -715,6 +732,21 @@
     </div>
   {/if}
 
+  {#if $isShowEditor}
```

Comment:
When the editor is opened the info panel should be hidden. Also worth considering:
- Pressing escape closes the editor instead of the asset viewer
- Hide the navbar actions while the editor is open and disable their shortcuts
- Disable shortcuts for previous/next asset

Reply:
is this the right way to disable shortcuts?
```js
<svelte:window
  use:shortcuts={[
    { shortcut: { key: 'a', shift: true }, onShortcut: toggleAssetArchive },
    { shortcut: { key: 'ArrowLeft' }, onShortcut: () => isShowEditor?null:navigateAsset('previous') },
    ...etc
```